### DISCUSSION TITLE
2 new features implemented

### DISCUSSION THREAD

**Discussion by isbjoernen** [ivandokov/phockup#185] [Ideas]
Hi, 

I added 2 new features:

    parser.add_argument(
        '-a',
        '--add-orig-names',
        action='store_true',
        help="""\
Organize the files in selected format or using the default year/month/day format but
appending the original filenames (cleaned of spaces) to the newly constructed filenames.
""",
    )

    parser.add_argument(
        '-j',
        '--one-jpg-extension',
        action='store_true',
        help="""\
Original files with extensions jpg, jpeg or JPG are all stored with the same file
name extension, namely jpg
""",
    )

In case someone is interested I'm happy to send it to the developpers. Just let me know how ;)

Cheers,
Arndt

**Comment by ivandokov** [comment]
Hello @isbjoernen, thank you for the feature suggestions.
If you want to send the code to the main repository please submit two pull requests.
[Creating a pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request)

**Comment by andyylin** [comment]
Would love to see these 2 features. Can @isbjoernen submit these pull requests?
